src/invidious/routes/video_playback.cr

```diff
@@ -42,7 +42,7 @@ module Invidious::Routes::VideoPlayback
       headers["Range"] = "bytes=#{range_for_head}"
     end
 
-    client = make_client(URI.parse(host), region)
+    client = make_client(URI.parse(host), region, force_resolve = true)
```

Comment:
Same for all the `force_resolve = true` or I am missing something? Why is this set to true?

Reply:
That's a request to googlevideos so it should go through IPv6

Reply:
No. One should still be able to choose the IP protocol.

Reply:
Oops I messed up on my explanation there. Setting `force_resolve = true` just means that the request will be affected by the force_resolve option in the configs. 

https://github.com/iv-org/invidious/blob/4f9332f652eca71bb38c0079fbd3c23003805ced/src/invidious/yt_backend/connection_pool.cr#L66-L68

Which if I'm not mistaken is what we want for requests to YouTube and Google to circumvent blockage

As long as the config option is unset then those request can freely go through either IPv4 and IPv6.

Reply:
Ha ok I did not understood the code like this.